docs/hosting/scaling/queue-mode.md

```diff
@@ -119,7 +127,7 @@ You can view running workers and their performance metrics in n8n by select **Se
 
 When running n8n with queues, all the production workflow executions get processed by worker processes. This means that even the webhook calls get delegated to the worker processes, which might add some overhead and extra latency. However, the manual workflow executions still use the main process.
 
-Redis is used as the message broker, and the database is used to persist data, so access to both is required. Running a distributed system with this setup over SQLite isn't recommended.
+Redis acts as the message broker, and the database persists data, so access to both is required. Running a distributed system with this setup over SQLite isn't recommended.
```

Comment:
I would maybe suggest that this setup wiht SQLite isn't supported rather than saying it isn't recommended.